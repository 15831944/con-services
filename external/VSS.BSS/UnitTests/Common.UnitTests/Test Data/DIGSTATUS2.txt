<?xml version="1.0" encoding="UTF-8"?>
<digitalSwitch2:topic xmlns:digitalSwitch2="http://www.cat.com/em/feed/v2/DigitalSwitchStataus2" xmlns:feed="http://www.cat.com/em/feed/v2/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v2/DIgitalSwitchStatus2 https://emfeed.cat.com/EMFeedFiles/schema/v2/DigitalSwitchStatus2.xsd">
   
<nextBuffer>
   <url>https://emfeed.rd.cat.com/EMFeedService/v2/feed/nighthawk/DIGSTATUS2/11974014</url>
   <moreData>true</moreData>
</nextBuffer>
<message>
      <header>
         <messageId>12096877</messageId>

         <masterMsgId>5504319</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DATA_FEED_TEST08783</moduleCode>
         <moduleTime>2009-02-05 20:00:51</moduleTime>
         <receivedTime>2009-02-05 20:00:51</receivedTime>
      </header>
      <powerMode>0</powerMode>
      <pending>No</pending>
      <switch1Active>Yes</switch1Active>
      <switch2Active>No</switch2Active>
      <switch3Active>No</switch3Active>
      <switch4Active>No</switch4Active>
   </message>

<message>
      <header>
         <messageId>12096937</messageId>
         <masterMsgId>5504364</masterMsgId>
         <equipment>
            <serialNumber>-1CF08764</serialNumber>
            <make>CAT</make>

            <model>DFTEST_351</model>
            <nickname>NICK_351</nickname>
         </equipment>
         <moduleCode>-DATA_FEED_TEST08764</moduleCode>
         <moduleTime>2009-02-05 19:12:51</moduleTime>
         <receivedTime>2009-02-05 19:12:51</receivedTime>

      </header>
      <powerMode>1</powerMode>
      <pending>Yes</pending>
      <switch1Active>No</switch1Active>
      <switch2Active>Yes</switch2Active>
      <switch3Active>No</switch3Active>
      <switch4Active>No</switch4Active>
   </message>
   <message>
      <header>
         <messageId>11971925</messageId>
         <masterMsgId>5491887</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>

            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>
         </equipment>
         <moduleCode>-DQ0200020816Q1</moduleCode>
         <moduleTime>2009-02-05 19:12:51</moduleTime>
         <receivedTime>2009-02-05 19:12:51</receivedTime>

      </header>
      <powerMode>2</powerMode>
      <pending>No</pending>
      <switch1Active>No</switch1Active>
      <switch2Active>No</switch2Active>
      <switch3Active>Yes</switch3Active>
      <switch4Active>No</switch4Active>
   </message>
   <message>
      <header>
         <messageId>11971925</messageId>
         <masterMsgId>5491887</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>

            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>
         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2009-02-05 19:12:51</moduleTime>
         <receivedTime>2009-02-05 19:12:51</receivedTime>

      </header>
      <powerMode>3</powerMode>
      <pending>Yes</pending>
      <switch1Active>No</switch1Active>
      <switch2Active>No</switch2Active>
      <switch3Active>No</switch3Active>
      <switch4Active>Yes</switch4Active>
   </message>
</digitalSwitch2:topic>

