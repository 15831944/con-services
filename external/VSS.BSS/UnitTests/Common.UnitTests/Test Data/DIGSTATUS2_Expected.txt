<?xml version="1.0" encoding="utf-8"?><ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10"><NHDataWrapper><Data z:Id="i1" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>12096877</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T20:00:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08783</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>0</d3p1:PowerMode><d3p1:SourceMsgID>5504319</d3p1:SourceMsgID><d3p1:Switch>1</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i2" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>12096877</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T20:00:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08783</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>0</d3p1:PowerMode><d3p1:SourceMsgID>5504319</d3p1:SourceMsgID><d3p1:Switch>2</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i3" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>12096877</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T20:00:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08783</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>0</d3p1:PowerMode><d3p1:SourceMsgID>5504319</d3p1:SourceMsgID><d3p1:Switch>3</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i4" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>12096877</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T20:00:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08783</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>0</d3p1:PowerMode><d3p1:SourceMsgID>5504319</d3p1:SourceMsgID><d3p1:Switch>4</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i5" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>397</d3p1:AssetID><d3p1:DebugRefID>12096937</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08764</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>1</d3p1:PowerMode><d3p1:SourceMsgID>5504364</d3p1:SourceMsgID><d3p1:Switch>1</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i6" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>397</d3p1:AssetID><d3p1:DebugRefID>12096937</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08764</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>1</d3p1:PowerMode><d3p1:SourceMsgID>5504364</d3p1:SourceMsgID><d3p1:Switch>2</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i7" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>397</d3p1:AssetID><d3p1:DebugRefID>12096937</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08764</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>1</d3p1:PowerMode><d3p1:SourceMsgID>5504364</d3p1:SourceMsgID><d3p1:Switch>3</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i8" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>397</d3p1:AssetID><d3p1:DebugRefID>12096937</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DATA_FEED_TEST08764</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>1</d3p1:PowerMode><d3p1:SourceMsgID>5504364</d3p1:SourceMsgID><d3p1:Switch>4</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i9" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200020816Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>2</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>1</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i10" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200020816Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>2</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>2</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i11" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200020816Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>2</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>3</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i12" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200020816Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>2</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>4</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i13" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>3</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>1</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i14" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>3</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>2</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i15" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>3</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>3</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i16" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataDigitalSwitchStatus" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>396</d3p1:AssetID><d3p1:DebugRefID>11971925</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:EventUTC>2009-02-05T19:12:51</d3p1:EventUTC><d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:IsActive>false</d3p1:IsActive><d3p1:Pending>false</d3p1:Pending><d3p1:PowerMode>3</d3p1:PowerMode><d3p1:SourceMsgID>5491887</d3p1:SourceMsgID><d3p1:Switch>4</d3p1:Switch><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper></ArrayOfNHDataWrapper>